pkg/git.go

```diff
@@ -246,7 +250,12 @@ func (p *GitPackage) Install(ctx context.Context, name, dir, version string) (st
 		return cmd
```

Comment:
I believe that you could discard output from the `gitCmd` if `GitQuiet` is set. As a bonus, if you use a bytes.Buffer, this could be printed if an error occurs. 

We have control on the stdout and stderr streams, thus we can silence the git output without relying on git implementation details (and reduce if/else on specific commands).

```
...
out := os.StdOut
outErr := os.Stderr
if GitQuiet {
    out = &bytes.Buffer{}
    outErr := &bytes.Buffer{}
}
cmd.Stdout = out
cmd.Stderr = outErr
```

if you do not want to keep output at all, set the streams to `nil`.

Reply:
Changed to discard both stdout/stderr to make it truly silent.